Makefile

```
@@ -32,7 +33,15 @@ createsuperuser:
 	$(MANAGE) createsuperuser
 
 
-freshstart: down build up
+rebuild:
+	$(MAKE) down
+	$(MAKE) up
```

I'm a little confused... should this be this still?
```suggestion
rebuild: down build up
```

---

Whoops yup! I was just testing that `$(MAKE)` syntax.